docs/property-graph-model.adoc

```diff
@@ -0,0 +1,75 @@
+:numbered:
+
+= Property Graph Model
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+Cypher is a graph query language, but it doesn't operate on any arbitrary definition of graph; it operates on a Property Graph.
```

Comment:
"Cypher is a graph query language, but it doesn't operate on any arbitrary definition of graph; it operates on a Property Graph."
->
"Cypher is a graph query language which operates on a _property graph_."